presto/presto.go

```diff
@@ -1123,15 +1119,15 @@ type rowFieldName struct {
 
 type typeSignature struct {
 	RawType   string         `json:"rawType"`
-	Arguments []typeArgument `json:"typeArguments"`
+	Arguments []typeArgument `json:"arguments"`
 }
 
 type typeKind string
 
 const (
-	KIND_TYPE       = typeKind("TYPE")
-	KIND_NAMED_TYPE = typeKind("NAMED_TYPE")
-	KIND_LONG       = typeKind("LONG")
+	KIND_TYPE       = typeKind("TYPE_SIGNATURE")
+	KIND_NAMED_TYPE = typeKind("NAMED_TYPE_SIGNATURE")
+	KIND_LONG       = typeKind("LONG_LITERAL")
```

Comment:
The names of these type argument "kinds" are apparently different between Trino and Presto (determined experimentally).